### PR TITLE
Page redesign

### PR DESCRIPTION
**Pull request by Henrik0x7F**
Notable changes:
- New style and color palette
- Removed Header. Replaced with bigger navbar
- Added download page
- Added social page linking to the discord server
- All markdown content now lies in a separate wiki directory accessible from wiki/main.md

### CODE REVIEW

File: src/components/Button.tsx [hunk 1]
```diff
@@ -2,29 +2,41 @@ import * as React from 'react'
...
 
 const StyledButton = styled.div`
+  font-size: 20px;
   display: inline-flex;
-  padding: ${getEmSize(6)}em;
   cursor: pointer;
-  border-radius: ${getEmSize(6)}em;
-  background-color: rgb(255, 229, 0);
-  box-shadow: rgba(0, 0, 0, 0.5) 1px 4px 13px 0px;
+  border-radius: 1000px;
```
Comment (Tarnadas) [inline src/components/Button.tsx]: You can instead use 50%, which should be more clear

File: src/components/Button.tsx [hunk 2]
```diff
@@ -2,29 +2,41 @@ import * as React from 'react'
 import styled from '@emotion/styled'
 
 import { getEmSize } from '../styles/mixins'
+import { colors } from '../styles/variables'
 import { Link, StaticQuery, graphql } from 'gatsby'
 
 const StyledButton = styled.div`
+  font-size: 20px;
```
Comment (Tarnadas) [inline src/components/Button.tsx]: Always use rem instead of px

File: src/components/Footer.tsx [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+const StyledFooter = styled.div`
+  font-family: Helvetica, sans-serif;
+  margin: 0;
+  border-style: solid;
+  border-color: ${colors.gray.copy};
+  border-width: 0;
+  border-top-width: 1px;
+  box-shadow: inset 0 5px 15px 0px ${colors.gray.copy};
+  padding-top: 10px;
```
Comment (Tarnadas) [inline src/components/Footer.tsx]: rem

File: src/components/MainNavigation.tsx [hunk 4]
```diff
@@ -0,0 +1,48 @@
+import * as React from 'react'
+
+import Navigation from './Navigation'
+
+
+interface MainNavigationPro {
```
Comment (Tarnadas) [inline src/components/MainNavigation.tsx]: You don't necessarily have to define empty prop interfaces. Just pass {} as props

File: src/components/Navigation.tsx [hunk 5]
```diff
@@ -8,15 +8,26 @@ import { getEmSize } from '../styles/mixins'
...
-  background-color: ${colors.brand};
-  display: flex;
+  display: block;
+  box-sizing: border-box;
+  border-style: solid;
+  border-width: 1px;
+  border-color: ${colors.gray.copy};
+  background-color: ${colors.white};
   align-items: center;
+  font-size: 18px;
```
Comment (Tarnadas) [inline src/components/Navigation.tsx]: Rem

File: src/components/NavigationLink.tsx [hunk 6]
```diff
@@ -2,17 +2,29 @@ import * as React from 'react'
...
   url: string
 }
 
 const StyledNavigationLink = styled.div`
-  margin: 0 ${getEmSize(12)}rem;
+  display: inline-block;
   a {
-    color: white;
+    outline: 0;
+    padding: 25px;
```
Comment (Tarnadas) [inline src/components/NavigationLink.tsx]: Rem

File: src/components/Button.tsx (hunk 1 above)
Comment (Henrik0x7F) [reply src/components/Button.tsx]: > You can instead use 50%, which should be more clear

Sadly this doesn't work. Using a big border-radius is the workaround pretty much everyone uses

**Pull request merged**